apache-jena-osgi/jena-osgi/src/main/resources/META-INF/services/org.apache.jena.system.JenaSubsystemLifecycle

```diff
@@ -1,4 +1,4 @@
-org.apache.jena.system.InitJenaCore
+org.apache.jena.sys.InitJenaCore
```

Comment:
Ouch! Yes, the initialization services classes will be impacted.

That means any customer extensions change but an intermediate subclass of JenaSubsystemLifecycle in the new "systems" package should provide a bridge.

Or we can be honest and announce to custom extensions there are changes they need to be aware of.

Reply:
+1 to honesty. I hear tell it's the best policy :) In all seriousness, the alternative seems to me to build up debt.

Reply:
I hope this debt isn't too bad because it can be signalled via deprecation. As ever, there is a balance.

The services filename ATM is `org.apache.jena.system.JenaSubsystemLifecycle`. I'm happy to and have a interface `...system.JenaSubsystemLifecycle` to make the transition not too abrupt.

(
Speculative thought, maybe trying too hard:

Can we actually move initialization all to jena-arq and just leave `Serializer` behind? `InitARQ` would call to be non-lifecycle managed initialization code in jena-core. That fixes the jena-core/jena-arq relationship. This may be less debt in the sense that it is a relationship the code depends on anyway. 

People using only jena-core aren't impacted - we actively discourage that but it happens. jena-core initialization happens via services currently but it can be dropped as it is using "services" to call itself, so that flexibility is unnecessary and it can just call itself in a fixed fashion.
)